pkg/cluster/ignition/ignition.go

```diff
@@ -91,12 +102,16 @@ func AddUser(ign *igntypes.Config, u *User) error {
 func AddGroup(ign *igntypes.Config, g *Group) error {
 	for _, pg := range ign.Passwd.Groups {
 		if pg.Name == g.Name {
-			return fmt.Errorf("A group with name %s is already defined", g.Name)
+			return errors.NewAlreadyExists(schema.GroupResource{
```

Comment:
What's the impetus for moving to this error library?

Reply:
I felt it was a cleaner way to check the error response for "already exists".  I will switch it back and scan the error text instead.